docs/tailwind-v4-guide.md

```
@@ -95,7 +95,7 @@ npm install tailwindcss@next @tailwindcss/vite@next
 - `blur-sm` → `blur-xs`
 - `bg-gradient-*` → `bg-linear-*`
 
-### Default Behavior Changes
+### Default Behaviour Changes
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)